nfpm.go

```diff
@@ -263,6 +263,7 @@ type Overridables struct {
 
 // RPM is custom configs that are only available on RPM packages.
 type RPM struct {
+	RPMArch     string       `yaml:"rpm_arch,omitempty" jsonschema:"title=architecture in rpm nomenclature"`
```

Comment:
```go
Arch     string       `yaml:"arch,omitempty" jsonschema:"title=architecture in rpm nomenclature"`
```

Reply:
Oh yeah, I thought `Arch` was in `Overridables` but it actually isn't.